app/schema.py

```diff
@@ -55,9 +84,88 @@ def mutate(root, info, contact, password):
         return LoginUser(logged_in=True, token=make_hash(user))
 
 
+class SendFriendRequest(graphene.Mutation):
+    class Arguments:
+        contact_receiver = graphene.String()
+    ok = graphene.Boolean()
+    user_receiver = graphene.Field(UserType)
+    friend_request = graphene.Field(FriendRequestType)
+
+    @staticmethod
+    @login_required
+    def mutate(root, info, contact_receiver):
+        try:
+            user_receiver = User.objects.get_by_natural_key(contact_receiver)
+
+        except User.DoesNotExist:
+            return SendFriendRequest(ok=False, user_receiver=None, friend_request=None)
```

Comment:
Put message as `User does not exist` or something like that

Reply:
Okay